p2p/addressbook/addrbook.go

```diff
@@ -305,8 +306,11 @@ func (a *AddrBook) AddressCache() []*AddrInfo {
 }
 
 // BootstrapAddressCache run AddressCache and add Config.AnchorPeersCount addresses from anchor peers.
-func (a *AddrBook) BootstrapAddressCache() []*AddrInfo {
-	addresses := a.AddressCache()
+func (a *AddrBook) BootstrapAddressCache() (addresses []*AddrInfo) {
+	addresses = a.AddressCache()
+	defer func() {
+		addresses = util.UniqueSliceStringer(addresses)
+	}()
```

Comment:
i think this is really weird way to do it. please just apply this function to addresses where they are returned